packages/scipy/patches/make_f2c_subroutines_void.patch

```diff
@@ -0,0 +1,374 @@
+diff --git a/scipy/sparse/linalg/dsolve/SuperLU/SRC/.cgsrfs.c.swp b/scipy/sparse/linalg/dsolve/SuperLU/SRC/.cgsrfs.c.swp
```

Comment:
Can you add some documentation to this patch? Are these changes something that can be done automatically?